### PR TITLE
Improve multiline answer alignment with input checkboxes

### PR DESCRIPTION
**Pull request by AChnki**
I changed the label css-styling for better alignment of multiline answer text with the input checkboxes.

I'm a css-newbie, so maybe there is a better or cleaner solution. Feel free to commit.

current alignment:
default

improved alignment - QType kprim:
QType_0

improved alignment - QType sc:
QType_2

improved alignment - QType mc:
QType_1

### CODE REVIEW

File: src/multiple_choice/card/css.css [hunk 1]
```diff
@@ -21,6 +21,8 @@ table {
 }
 
 label {
+  display: inline-block;
+  vertical-align:middle;
```
Comment (3ter) [inline src/multiple_choice/card/css.css:25]: Why no space 😜? Just format the whole doc (if there are other missing spaces for a common formatting).

File: src/multiple_choice/config.py [hunk 2]
```diff
@@ -51,8 +51,8 @@
 
 # default configurations
 # TODO: update version number before release
-default_conf_local = {'version': "2.7.1"}
-default_conf_syncd = {'version': "2.7.1"}
+default_conf_local = {'version': "2.7.2"}
```
Comment (3ter) [inline src/multiple_choice/config.py:54]: Incrementing the bugfix version is okay, if you would describe the current behaviour as unintended. I would rather have it as a feature version (`2.8.0`) but both is totally fine with me.

**Review by 3ter — APPROVED** [review]
Thank you for your contribution!

I haven't tried it, but I'm interested how this will work with code and tables as answer options.

File: src/multiple_choice/card/css.css (hunk 1 above)
Comment (AChnki) [reply src/multiple_choice/card/css.css:25]: Sorry, I don't get your point here. 🤔
Where are spaces missing?

**Comment by AChnki** [comment]
> I haven't tried it, but I'm interested how this will work with code and tables as answer options.

Here are some screenshots with QType=2:

old css-styling:
grafik

new proposed css-styling:
grafik

But it seems that tables within the choices are not supported or I made a mistake-> #97

**Comment by 3ter** [comment]
Yes, you're not mistaken, thanks for the detailed bug report.

**Comment by zjosua** [comment]
Thank you for the contribution @AChnki.
I also prefer to have the minor version incremented in this case. Btw, the one you changed is called patch verison according to the [semver documentation](https://semver.org/) I use.

**Pull request merged**